internal/pkg/utils/testhelper/terminal/terminal.go

```diff
@@ -8,7 +8,7 @@ import (
 )
 
 const sendDelay = 20 * time.Millisecond
-const expectTimeout = 5 * time.Second
+const expectTimeout = 10 * time.Second
```

Comment:
Dlhsi timeout, ... lokalne sa mi stalo, ze nacitanie `dev-branches` trvalo `>5s` a CLI dialog na to cakal -> spadlo to.